cmd/relayer/main.go

```diff
@@ -64,6 +72,27 @@ var (
 			Value: "info",
 			Usage: "Set log level: one of [error|warn|info|debug]",
 		},
+		&cli.BoolFlag{
+			Name:  flagWithNetwork,
+			Value: false,
+			Usage: "Run the relayer with p2p network capabilities",
+		},
```

Comment:
If we want the defaults to reflect the most common use case, should the above flag be true by default?